src/lib389/cli/dscontainer

```diff
@@ -462,6 +477,9 @@ container host.
     parser.add_argument('-r', '--runit',
                         help="Actually run the instance! You understand what that means ...",
                         action='store_true', default=False, dest='runit')
+    parser.add_argument('-s', '--stop',
```

Comment:
How do you intend to call "stop"?

Reply:
I think a `stop()` function is mostly useful for running dscontainer during a container build, then shutting it down gracefully (though container debugging could be another use case).  Specifically there are some initialization things that happen only on first run (but must happen).  Plus, it's possible someone would want to extend the image and build-in all of their configuration up-front during the build.   This wouldn't be a generally applicable image at that point, but some people may want a custom image for personal use.  If so, being able to gracefully stop would be nice.  Here is where I would use it: https://github.com/JeffersonLab/dirsrv/blob/77f3ead9d19bb990ce0814b3f7960c8d8c7674eb/scripts/once.sh#L29

Reply:
Also worth mentioning it really only makes sense to have `stop()` if dscontainer is NOT running as init (PID 1).  In my case I run a bash script [docker-entrypoint.sh](https://github.com/JeffersonLab/dirsrv/blob/main/scripts/docker-entrypoint.sh) as init to give me the most flexibility and I think this is a common case for development and testing.  I think bash does reap zombies, but may not forward signals though.   I was pondering how best to handle PID 1 here: https://github.com/389ds/389-ds-base/discussions/6058#discussioncomment-8323600